tests/cephfs/cephfs_mirroring/test_cephfs_mirror_ha_conversion.py

```diff
@@ -0,0 +1,303 @@
+        log.info(f"MDS host list 1 {mds_hosts_1}")
+        source_clients[0].exec_command(
+            sudo=True,
+            cmd=f"ceph orch apply cephfs-mirror --placement='2 {mds_hosts_1}'",
+        )
+        fs_util_ceph1.validate_services(source_clients[0], "cephfs-mirror")
+        peer_uuid = fs_mirroring_utils.get_peer_uuid_by_name(
+            source_clients[0], source_fs
+        )
+        if len(mds_nodes) > 2:
+            cephfs_mirror_node = mds_nodes[1:3]
+        else:
+            cephfs_mirror_node = mds_nodes
+        fs_mirroring_utils.add_files_and_validate(
+            source_clients,
+            kernel_mounting_dir_1,
+            subvol1_path,
+            fuse_mounting_dir_1,
+            subvol2_path,
+            cephfs_mirror_node,
+            source_fs,
+            "ha_service_22",
+            snap_count=1,
+        )
+
+        out, rc = fs_mirroring_utils.list_and_verify_remote_snapshots_and_data_checksum(
+            target_clients[0],
+            "client.admin",
+            subvol2_path,
+            source_clients[0],
+            fuse_mounting_dir_1,
+            target_fs,
+        )
+        if not out:
+            raise CommandFailed(f"{rc}")
+        out, rc = fs_mirroring_utils.list_and_verify_remote_snapshots_and_data_checksum(
+            target_clients[0],
+            "client.admin",
+            subvol1_path,
+            source_clients[0],
+            kernel_mounting_dir_1,
+            target_fs,
+        )
+        if not out:
+            raise CommandFailed(f"{rc}")
+        return 0
+    except Exception as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    finally:
+        log.info("Clean up the system")
+        log.info("Delete the snapshots")
+        snapshots_to_delete = [
+            f"{kernel_mounting_dir_1}{subvol1_path}.snap/snap_k1_single_service*",
+            f"{kernel_mounting_dir_1}{subvol1_path}.snap/snap_k1_ha_service_*",
+            f"{fuse_mounting_dir_1}{subvol2_path}.snap/snap_f1_single_service*",
+            f"{fuse_mounting_dir_1}{subvol2_path}.snap/snap_f1_ha_service_*",
+        ]
+        for snapshot_path in snapshots_to_delete:
+            source_clients[0].exec_command(
+                sudo=True, cmd=f"rmdir {snapshot_path}", check_ec=False
+            )
+
+        log.info("Unmount the paths")
+        paths_to_unmount = [kernel_mounting_dir_1, fuse_mounting_dir_1]
+        for path in paths_to_unmount:
+            source_clients[0].exec_command(sudo=True, cmd=f"umount -l {path}")
+
+        log.info("Remove paths used for mirroring")
+        paths_to_remove = [subvol1_path, subvol2_path]
+        for path in paths_to_remove:
+            fs_mirroring_utils.remove_path_from_mirroring(
+                source_clients[0], source_fs, path
+            )
+
+        log.info("Destroy CephFS Mirroring setup.")
+        fs_mirroring_utils.destroy_cephfs_mirroring(
```

Comment:
We are seeing error in logs at this step, "Error EINVAL: filesystem cephfs_rem is not mirrored", Can you please evaluate?